doc/source/servers/huggingface.md

```diff
@@ -8,12 +8,13 @@ We also support the high performance optimizations provided by the [Transformer
 
 The parameters that are available for you to configure include:
 
-| Name | Description |
-| ---- | ----------- |
-| `task` | The transformer pipeline task |
-| `pretrained_model` | The name of the pretrained model in the Hub |
+| Name                   | Description                                                         |
+|------------------------|---------------------------------------------------------------------|
+| `modelUri`             | The artefacts location of a custom HuggingFace model                |
```

Comment:
I would not include this one, as it goes into a different part of the `SeldonDeploment` resource (i.e. not outside of the `parameters` field).

Reply:
Yeah, that's a good point. Removing